dask/dataframe/utils.py

```diff
@@ -160,3 +160,7 @@ def get_categories(df):
     if iscategorical(df.index.dtype):
         result['.index'] = df.index.categories
     return result
+
+
+def is_frame_or_series(x):
+    return isinstance(x, (pd.Series, pd.DataFrame))
```

Comment:
No mandatory reason. A background is ``utils`` already have shortening type check func like ``iscategorical``.

Changed to use ``isinstance``.